dll/win32/kernel32/wine/res.c

```diff
@@ -142,7 +142,11 @@ static HRSRC find_resourceW( HMODULE hModule, LPCWSTR type, LPCWSTR name, WORD l
  */
 HRSRC WINAPI FindResourceExA( HMODULE hModule, LPCSTR type, LPCSTR name, WORD lang )
 {
+#ifdef __REACTOS__
+    TRACE( "%p %08x %s %04x\n", hModule, (ULONG)type, debugstr_a(name), lang );
```

Comment:
What exactly is the value you're seeing a crash with here? debugstr_a should correctly handle `NULL` and `MAKEINTRESOURCE` pointers, so it shouldn't crash unless the value passed is truly invalid.

Reply:
Unfortunately I couldn't reproduce this myself. Page fault exception was encountered by @mborgerson reliably on `FindResourceExW` call, I asked him to get debug output for `type` and `name` pointers.

Reply:
Ok I removed the second commit for now, since it's not well debugged yet, so this PR wouldn't stall forever. :slightly_smiling_face: